### PR TITLE
PR for homepage 

### PR DESCRIPTION
**Pull request by narharikale**
Added carousel .
added slider.

deploy preview- https://deploy-preview-2--epicplay.netlify.app/

not responsive yet

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -33,6 +33,7 @@ button, input, textarea {
 
 #App.App{
```
Comment (horsemaker) [inline src/App.css:34]: Why `#` + `.`? 🤔

File: src/backend/db/categories.js [hunk 2]
```diff
@@ -8,20 +8,22 @@ import { v4 as uuid } from "uuid";
...
-      "Backend Development is also known as server-side development. It is everything that the users don't see and contains behind-the-scenes activities that occur when performing any action on a website.",
+    categoryName: "Sci-Fi",
+  },
+  {
+    _id: uuid(),
+    categoryName: "Animated",
+  },
+  {
+    _id: uuid(),
+    categoryName: "Documentry",
```
Comment (horsemaker) [inline src/backend/db/categories.js:27]: Typo. Should be `Documentary`.

**Review by horsemaker — APPROVED** [review]
- Code looks super clean!
- Liked the use of Slider & Carousel, works super fine!

**Review by MiheerTamkhane — APPROVED** [review]
For me code is readable and also working smooth on browser.
Nice work Good to merge

File: src/App.css (hunk 1 above)
Comment (narharikale) [reply src/App.css:34]: be specific😐

File: src/backend/db/categories.js (hunk 2 above)
Comment (narharikale) [reply src/backend/db/categories.js:27]: yes , thanks

**Pull request merged**